destination/destination_integration_test.go

```diff
@@ -0,0 +1,395 @@
+// Copyright © 2022 Meroxa, Inc. & Yalantis
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package destination
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/matryer/is"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+
+	"github.com/conduitio-labs/conduit-connector-mongo/config"
+	"github.com/conduitio-labs/conduit-connector-mongo/destination/writer"
+)
+
+const (
+	// set the directConnection to true in order to avoid the known hostname problem.
+	testURI              = "mongodb://localhost:27017/?directConnection=true"
+	testDB               = "test"
+	testCollectionPrefix = "test_coll"
+
+	// next consts will be used for test models.
+	testIDFieldName = "_id"
+	testID          = "6384c3e48740eb54f858bde5"
+
+	testTextFieldName = "textfield"
+	testTextData      = "some text data"
+
+	testNumberFieldName  = "numberfield"
+	testNumberData       = 1234
+	testNumberDataChange = 5678
```

Comment:
Let's remove the `testID`, `testTextData`, `testNumberData` and `testNumberDataChange`. And let's generate the data using the `gofakeit` package, like in the other tests, to be consistent.

Reply:
fixed